modules/openapi-generator/src/main/resources/JavaJaxRS/beanValidation.mustache

```diff
@@ -1 +1 @@
-{{#required}}@NotNull {{/required}}{{^isPrimitiveType}}{{^isDate}}{{^isDateTime}}{{^isString}}{{^isFile}}@Valid {{/isFile}}{{/isString}}{{/isDateTime}}{{/isDate}}{{/isPrimitiveType}}{{>beanValidationCore}}
\ No newline at end of file
+{{#required}}@NotNull {{/required}}{{#isContainer}}{{^isPrimitiveType}}{{^isDate}}{{^isDateTime}}{{^isString}}{{^isFile}}{{^isEnumOrRef}}@Valid {{/isEnumOrRef}}{{/isFile}}{{/isString}}{{/isDateTime}}{{/isDate}}{{/isPrimitiveType}}{{/isContainer}}{{>beanValidationCore}}
```

Comment:
> {{#isContainer}}{{^isPrimitiveType}}{{^isDate}}{{^isDateTime}}{{^isString}}{{^isFile}}{{^isEnumOrRef}}@Valid {{/isEnumOrRef}}{{/isFile}}{{/isString}}{{/isDateTime}}{{/isDate}}{{/isPrimitiveType}}{{/isContainer}}

let's try to simplify the logic here

can you tell me what type(s) need `@Valid` ?

Reply:
The types you want to remove `@Valid` are

`MyEnumClass`: Class of type Enum that is not inline (as I wrote in Issue, `@Valid` is not given in case of inline. This behavior is preferred)
List<`MyEnumClass`>: List of Enums using it

Reply:
The types I want to remove @Valid from are:

MyEnumClass: A class of type Enum that is not inline (as I mentioned in the issue, @Valid is not applied in the case of inline definitions. This behavior is preferred).
List<MyEnumClass>: A list of Enums.

Reply:
what about other way around - what needs the @Valid annotation?

Reply:
This action is understood to be an additional measure for the main issue at https://github.com/OpenAPITools/openapi-generator/pull/14435. In my environment, the performance of the Hibernate Validator related to Enum values has significantly decreased. Therefore, I would like to implement measures only for Enum values in this case. At the moment, I have no other requests, such as wanting to add @Valid to other items.

Reply:
I simply wanted to simplify the logic here a bit to make it easier to maintain moving forward. (not saying your approach/change is right or wrong)

Reply:
Yes, it is. I too would like to achieve this in a simpler way.

Reply:
What about list of integers? or list of models? 

Do these need @Valid?

Reply:
I don't think `@Valid` is needed for PrimitiveType (and its wrapper types) and Enum's List.

Reply:
Is there any progress on this one? @wing328

Reply:
I would like to implement this.
> I don't think @Valid is needed for PrimitiveType's (its wrapper types) List and Enum's List.

Reply:
@wing328

Reply:
when you've time this week, can you please PM me via Slack to further discuss this?

https://join.slack.com/t/openapi-generator/shared_invite/zt-12jxxd7p2-XUeQM~4pzsU9x~eGLQqX2g